### PR TITLE
Wk 24/detail panel

### PR DESCRIPTION
**Pull request by BennievanderWel**
The on marker click bug is still in. I'm planning to take that up this week together with some help of more experienced Leaflet users. I will fix that in another branch.

### CODE REVIEW

**Comment by BennievanderWel** [comment]
@RRMoelker It seems like the api changed a bit. I've adjusted the processing of the api accordingly and added some dev feedback in erroneous situations. Can you have a look?

**Comment by RRMoelker** [comment]
I'm getting some errors in the console when clicking on a marker. Can you take a look?

```
Warning: Failed prop type: The prop `status` is marked as required in `SVGIcon`, but its value is `undefined`.
    in SVGIcon (at DetailPanel.js:43)
    in DetailPanel (at Map.js:127)
    in div (at Map.js:112)
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in styled.div (at Map.js:111)
    in Map (at DashboardPage.js:9)
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in styled.div (at DashboardPage.js:8)
    in Unknown (created by Route)
    in Route (at App.js:20)
    in Switch (at App.js:17)
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in styled.div (at App.js:16)
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in styled.div (at App.js:14)
    in App (at src/index.js:15)
    in Router (created by BrowserRouter)
    in BrowserRouter (at src/index.js:12)
    in ThemeProvider (at src/index.js:11)
```

**Comment by RRMoelker** [comment]
I'm also getting `index.js:1446 Unable to resolve a spot type or status` when clicking on a marker.

**Comment by RRMoelker** [comment]
The map is jumping to the wrong location when clicking certain markers....

File: src/components/detailPanel/DetailPanel.js [hunk 1]
```diff
@@ -1,18 +1,108 @@
...
+                <td>{jaar_blackspot_lijst}</td>
+              </tr>
+            </tbody>
+          </DataTable>
+          <h3>Maatregelen</h3>
+          <DataTable>
+            <tbody>
+              <tr>
+                <td>Actiehouder</td>
+                <td>WBA</td>
```
Comment (RRMoelker) [inline src/components/detailPanel/DetailPanel.js]: Use API value

**Pull request merged**